api/net/Areas/Subscriber/Controllers/ContentController.cs

```diff
@@ -327,5 +327,38 @@ public async Task<IActionResult> ShareAsync(long contentId, int colleagueId, int
         await _kafkaMessenger.SendMessageAsync(_kafkaOptions.NotificationTopic, $"notification-{notification.Id}-test", request);
         return new JsonResult(new NotificationModel(notification, _serializerOptions));
     }
+
+    /// <summary>
+    /// Send the notification to the specified email address.
+    /// </summary>
+    /// <param name="contentId"></param>
+    /// <param name="email"></param>
+    /// <param name="notificationId"></param>
+    /// <returns></returns>
+    [HttpPost("{contentId}/share/email")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(NotificationModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Notification" })]
+    public async Task<IActionResult> ShareEmailAsync(long contentId, string email, int notificationId)
+    {
+        var notification = _notificationService.FindById(notificationId) ?? throw new NoContentException();
+
+        var username = User.GetUsername() ?? throw new NotAuthorizedException("Username is missing");
+        var user = _userService.FindByUsername(username) ?? throw new NotAuthorizedException("User does not exist");
+        var subscriber = _userService.FindByEmail(email).FirstOrDefault() ?? throw new NotAuthorizedException("Subscriber does not exist");
```

Comment:
Make this return a BadRequest instead of a 403.  InvalidOperationException should do this.